.github/workflows/macos.yml

```diff
@@ -29,9 +32,7 @@ jobs:
             -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache)
         cmake --build build --parallel 2
 
-        cd build
-        ctest --output-on-failure
-        cd ..
+        ctest --test-dir build --output-on-failure
 
   # Build libamrex and all tests
```

Comment:
@WeiqunZhang do we need that 2nd test? I am ok to only build one macOS build with EB on and cover non-EB code paths on other platforms.

Reply:
Ah, universal builds need universal dependencies (like MPI) as well. In that case I keep two runs here.